CHANGES.rst

```diff
@@ -6,6 +6,12 @@ General
 
 - Add TweakReg submodule. [#267]
 
+ramp_fitting
+~~~~~~~~~~~~
+
+- Move the CHARGELOSS read noise variance recalculation from the JWST step
+  code to the C extension to simplify the code and improve performance.[#275]
+
```

Comment:
```suggestion
ramp_fitting
~~~~~~~~~~~~

- Move the CHARGELOSS read noise variance recalculation from the JWST step
  code to the C extension to simplify the code and improve performance.[#275]

```
#284 switched change log handling to `towncrier`, so this change log entry should be a file in `changes/`:
```shell
echo "[ramp_fitting] Move the CHARGELOSS read noise variance recalculation from the JWST step code to the C extension to simplify the code and improve performance." > changes/275.general.rst
```

(new PRs will have these instructions in the checklist)